docs/_includes/browsers.md

```diff
@@ -3,7 +3,7 @@ that you use the most up-to-date browser that's compatible with your operating s
 supported.
 
 * The new [Microsoft Edge](https://www.microsoft.com/edge)
-* Safari (latest version, Mac only)
+* [Safari](https://www.apple.com/safari) (latest version)
```

Comment:
Just double-checking -- do we officially support iOS now? I think that's what the Mac-only flag was intended to indicate -- that we only supported Safari on the Mac.

Reply:
The framework definitely supports iOS safari and we should be fixing bugs here if end users find them. There are multiple teams building these scenarios currently.

We do not support iOS native.